ol_sale_optional_product/models/sale_order_option.py

```diff
@@ -42,6 +42,8 @@ class SaleOrderOption(models.Model):
         store=True,
         precompute=True,
     )
+    mrp_bom_id = fields.Many2one("mrp.bom",string="BOM")
+    config_session_id = fields.Many2one("product.config.session",string="Config session")
 
     # END #########
     # METHODS #####
```

Comment:
If a user clicks the cart icon on the option line to add the line to the sale order, it should also populate the bom and session there to, can you verify that is happening?

Reply:
> If a user clicks the cart icon on the option line to add the line to the sale order, it should also populate the bom and session there to, can you verify that is happening?

Yes, I added code to pass these two new fields in this _get_values_to_add_to_order()